src/main/java/com/ebay/sonar/test/github/TestGithub.java

```diff
@@ -4,6 +4,15 @@
 
 	public void testJava(){
 		System.out.println("test");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS106)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AIndentationCheck)